CONTRIBUTING.md

```diff
@@ -0,0 +1,295 @@
+# Introduction
+
+Well hello there my dear reader! Thank you for considering contributing to the Fantomas project!<br />
+Fantomas is a bit of a complex project and it can be overwhelming to just dive in.
+In this guide, we will try to get you started, and express our expectations in terms of potential contributions.<br />
```

Comment:
nit: there's an inconsistency in style, sometimes `<br/>` is used and sometimes it's not; I think markdown format invites to have as less markup as possible so I think it's better to not have it (and replace it with a real EOL). At the end of the day, markdown is supposed to be friendly for both console readers and for browser ones (and I guess it was born from the point of view of having an easier-than-tags format).